plugins/gcloud/app/controllers/gcloud_controller.rb

```diff
@@ -45,6 +46,5 @@ def update_build(build)
     end
 
     build.save!
```

Comment:
can we do this to not raise/rescue and stay with the current pattern ?
`return "Failed to save build #{build.errors.full_messages}" unless build.save`